### PR TITLE
Add function for register

### PR DESCRIPTION
**Pull request by illicitonion**

### CODE REVIEW

File: org-cyf-itp/tooling/netlify/functions/submission-created.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    const response = await sheets.spreadsheets.values.append({
+      spreadsheetId: COURSE_TO_SPREADSHEET_ID[request.course],
+      // Trust that the user-supplied module exists.
+      // If it's wrong, we'll get a 400 from the API and pass it on to the user without much useful detail.
+      range: request.module,
+      valueInputOption: "RAW",
+      insertDataOption: "INSERT_ROWS",
+      resource: {
+        values: [
+          [request.givenName, request.familyName, request.email, new Date().toISOString(), request.course, request.module, request.day],
```
Comment (SallyMcGrath) [inline org-cyf-itp/tooling/netlify/functions/submission-created.js:86]: FYI, the reason I included two dates was that I wanted to easily colour the sign in red if the signing date was not within 1 day of the build date of the form, given that the site is rebuilt every night. This was my extremely slapdash/goodenough solution to someone going through and signing in retrospectively or speculatively without having to set up a dictionary of class dates to check against. 

How can you solve this problem?

File: common-theme/layouts/partials/register-attendance.html [hunk 2]
```diff
@@ -38,7 +38,7 @@ <h2 class="c-block__title e-heading__2" id="register">
   <input type="hidden" name="day" value="{{ $day }}">
   <input type="hidden" name="buildTime" value="{{ now.Format "2006-01-02T15:04:05" }}">
   <div><label for="givenName" class="is-invisible">Given Name</label>
-  <input type="text" name="name" id="givenName" autocomplete="given-name" placeholder="Given Name" required>
+  <input type="text" name="givenName" id="givenName" autocomplete="given-name" placeholder="Given Name" required>
```
Comment (SallyMcGrath) [inline common-theme/layouts/partials/register-attendance.html:41]: 🤦‍♀️ 🙇‍♀️

**Review by SallyMcGrath — APPROVED** [review]
non blocking but see comment

File: org-cyf-itp/tooling/netlify/functions/submission-created.js (hunk 1 above)
Comment (illicitonion) [reply org-cyf-itp/tooling/netlify/functions/submission-created.js]: For now, I've just preserved your existing solution :)

**Pull request merged**